frontend/src/Components.res

```diff
@@ -934,7 +934,24 @@ module Link = {
         []
       },
     })
-    <a className={Sx.make(sx)} ?href ?target> icon_svg text </a>
+    <a
+      className={Sx.make(sx)}
+      href
+      ?target
+      onClick={event =>
+        // https://github.com/MinimaHQ/safe-routing-blog-app-example/blob/master/02-multiple-types/src/Router.re
```

Comment:
Interesting read on how to make URLs safer: https://blog.minima.app/posts/2020/reasonml-safe-routing